server/test/run

```diff
@@ -13,26 +13,24 @@ unset _PBENCH_SERVER_CONFIG
 export LANG=C
 export LC_ALL=C
 
-export _server=$(dirname $(dirname $(realpath ${0})))
+function _log_exit {
+    echo "${1}" >&2
+    exit 1
+}
+
+export _server=$(dirname $(dirname $(realpath "${0}")))
 export _bdir=${_server}/bin
-test -d "${_bdir}" || exit 1
+test -d "${_bdir}" || _log_exit "ERROR: Missing \"${_bdir}\""
```

Comment:
OK, so this comes out something like `ERROR: Missing "server/bin"`, but we're doing a `-d` check not a `-e` check; so maybe something more like `ERROR: ${_bdir} missing or not a directory`??

Reply:
Fixed.